packages/bento-design-system/src/Icons/IconEye.tsx

```diff
@@ -4,7 +4,18 @@ import { svgIconProps } from "./svgIconProps";
 export function IconEye(props: IconProps) {
   return (
     <svg {...svgIconProps(props)}>
-      <path d="M12 4C6.545 4 1.887 7.393 0 12.182c1.887 4.789 6.545 8.182 12 8.182 5.454 0 10.113-3.393 12-8.182C22.113 7.392 17.454 4 12 4Zm0 13.636a5.457 5.457 0 0 1-5.455-5.454A5.457 5.457 0 0 1 12 6.727a5.457 5.457 0 0 1 5.454 5.455A5.457 5.457 0 0 1 12 17.636Zm0-8.727a3.268 3.268 0 0 0-3.273 3.273A3.268 3.268 0 0 0 12 15.454a3.268 3.268 0 0 0 3.273-3.272A3.268 3.268 0 0 0 12 8.909Z" />
+      <path
+        fill-rule="evenodd"
+        clip-rule="evenodd"
+        d="M12 9.82982C11.3962 9.82982 10.8171 10.0697 10.3902 10.4966C9.96324 10.9236 9.72339 11.5026 9.72339 12.1064C9.72339 12.7102 9.96324 13.2893 10.3902 13.7162C10.8171 14.1432 11.3962 14.383 12 14.383C12.6038 14.383 13.1829 14.1432 13.6098 13.7162C14.0367 13.2893 14.2766 12.7102 14.2766 12.1064C14.2766 11.5026 14.0367 10.9236 13.6098 10.4966C13.1829 10.0697 12.6038 9.82982 12 9.82982ZM8.78038 8.88682C9.63428 8.03293 10.7924 7.55322 12 7.55322C13.2076 7.55322 14.3657 8.03293 15.2196 8.88682C16.0735 9.74071 16.5532 10.8988 16.5532 12.1064C16.5532 13.314 16.0735 14.4721 15.2196 15.326C14.3657 16.1799 13.2076 16.6596 12 16.6596C10.7924 16.6596 9.63428 16.1799 8.78038 15.326C7.92649 14.4721 7.44678 13.314 7.44678 12.1064C7.44678 10.8988 7.92649 9.74071 8.78038 8.88682Z"
+        fill="#525E6F"
```

Comment:
as discussed on Slack, let's remove this fill so that it gets inherited from the svg props